### ISSUE TITLE
Add NFT tracking

### ISSUE THREAD

**Issue by ThorstenBux** [augmentmy-world/arStudio#11]
@kalwalt 
Where are we with this?

**Comment by kalwalt** [comment]
@ThorstenBux it is partially implemented. Actually you can select the Pinball marker  but it can't be loaded because parse a wrong path: it parse the patt.hiro path plus the .fset extension and so fails to load.

**Comment by kalwalt** [comment]
Pr #17 testing the NFT feature.

**Comment by kalwalt** [comment]
With the PR #17 we added support for NFT markers. For now we can only load the Pinball marker, and without the WebWorker. This will implemented in another Pull Request.

**Comment by kalwalt** [comment]
@ThorstenBux May i add the webWorker or is better to implement other features before? Like loading of patterns (NFT, Pattern...) ? I would also test, during these days, if the Test and Publish feature works or not...

**Comment by ThorstenBux** [comment]
Yes, sure go for it. :). Get the webWorker in that is important.
Regarding uploads of NFT I was thinking if we can use zipped versions then we could just upload one

**Comment by kalwalt** [comment]
> Yes, sure go for it. :). Get the webWorker in that is important.

Ok I will start to work on this feature. I colud also try to embedd the WebWorker inside the Component with a Blob object  like did recently for Ar.js. But probably better to start with a simpler approach for now.

> Regarding uploads of NFT I was thinking if we can use zipped versions then we could just upload one

In this case we should skip the problem of the format?

**Comment by kalwalt** [comment]
Trying to add a WebWorker with this PR https://github.com/augmentmy-world/arStudio/pull/34
